packages/stark-ui/src/modules/action-bar/components/action-bar.component.html

```diff
@@ -4,27 +4,27 @@
 		<div *ngFor="let action of actionBarConfig.actions; trackBy: trackAction" [id]="actionBarId + '-' + action.id"
 		(click)="onClick(action, $event)" class="stark-action-bar--action">
 			<ng-container *ngIf="action.isVisible !== false">
-				<button [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label" [disabled]="!action.isEnabled" mat-icon-button>
-					<mat-icon starkSvgViewBox [svgIcon]="action.iconSwitchFunction ? action.iconActivated : action.icon"></mat-icon>
+				<button color="primary" [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label" [disabled]="!action.isEnabled" mat-icon-button>
+					<mat-icon starkSvgViewBox [svgIcon]="action.iconSwitchFunction ? action.iconActivated : action.icon" class="stark-small-icon"></mat-icon>
 				</button>
 				<label class="action-label" [ngClass]="{'disabled': !action.isEnabled}" *ngIf="isExtended" translate>{{ action.label }}</label>
 			</ng-container>
 		</div>
 	</div>
 
 	<div *ngIf="mode === 'full' || alternativeActions" class="alt-actions">
-		<button class="extend-action-bar" mat-icon-button *ngIf="mode === 'full'" (click)="toggleExtendedActionBar()">
-			<mat-icon starkSvgViewBox svgIcon="dots-horizontal"></mat-icon>
+		<button class="extend-action-bar" color="primary" mat-icon-button *ngIf="mode === 'full'" (click)="toggleExtendedActionBar()">
```

Comment:
Why force setting primary color?
Maybe should we provide a new property in the interface to set a color? Which is an optional string and, if it is not set, we can set primary by default. What do you think about this? :blush:

Reply:
This is a feature coming from Angular Material, nothing added on our side.

https://material.angular.io/components/button/examples

Reply:
If we want a closest integration with Angular Material, I think we should keep the directive that they provide. Now I have extend the 3 default colors to add the one we were using before. Then the end user could use Angular Material button with our custom colors, I think that provide the most flexibility.

Imagine if we give up on the color attr and they write this:
<button color="primary" class="stark-alert" mat-raised-button>Primary or Alert?</button>
Then this button have 2 styles defined, so one should override the other

With this notation:
<button color="alert" mat-raised-button>Alert</button>
There is no confusion and we stay inline with Angular Material

Reply:
If we really want to override the color we can do it through a mixin that I have created, which use actually the angular material mixin for buttons which takes the color and the text color as argument

The code for end user will be:
.my-custom-button {
  @include stark-button-color(#fff, #222);
}

Reply:
@catlabs What do you think if we add a property in StarkAction ?
Now, we have something like this:
```typescript
export interface StarkAction {
  // ...
  isEnabled: boolean;
  label: string;
  // ...
}
```

But we could have something else like:
```typescript
export interface StarkAction {
  // ...
  isEnabled: boolean;
  label: string;
  buttonColor: string;    //    <-------
  // ...
}
```

Then, in the component, we could have this:
```html
<button color="action.buttonColor ? action.buttonColor : 'alert'" mat-raised-button>Alert</button>
```

What do you think about this ?
 
We continue to use Angular Material as it should be but we give more abilities to choose the color of the button.
I don't think we should use `@mixin` function to change the color of the button...

Reply:
Well yeah that's an idea, we talked about it with @christophercr actually, I ll do that and repush the change

Reply:
Mmm, but then we should think about what are the possible colors we would accept; any? just primary, accent and alert?

If we accept any, in case a new color is given, then it means that the developer should also "create" that color with the Angular Material mixins which at the end willl be the same effort that the one @catlabs mentioned above in order to override the color.

So, I don't know if this approach of having a buttonColor in the interface will make things easier or not

Reply:
I think you're right Christopher :blush:

We could do so
```typescript
{
  //... 
  buttonColor: "primary" | "alert" | "accent";
} 
```

What do you think about this?
And if they want more flexibility, they will have to play with `@mixin` function :blush:

Reply:
Well yeah that's the idea, they can either enter one of the seven colors we provide (primary, warn,accent,success,alert,alt,neutral,white) or a custom one, then they will have to play with the mixin